internal/provider/resource_settings.go

```diff
@@ -0,0 +1,136 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/infrahq/infra/api"
+)
+
+func resourceSettings() *schema.Resource {
+	return &schema.Resource{
+		Description: `Providers Infra organization settings.
+
+` + "`infra_settings`" + ` behaves differently than normal Terraform resources as settings are
+created with the organization. When a Terraform resource is created, settings automatically
+imported while no action is taken when the resource is deleted.
+
+-> This resource requires Infra server version 0.20.0 or higher.
```

Comment:
Can we replace this with the standard Client <> Server API version check we introduced in the `infra` CLI? The `infrahq/infra/api` client should do it automatically.

Reply:
These are not the same checks. The CLI check ensures the server isn't too far ahead relative to the client while this check ensures the server is at least some version.